cmd/vcert/result_writer.go

```diff
@@ -117,6 +122,106 @@ func (o *Output) AsPKCS12(c *Config) ([]byte, error) {
 	return bytes, nil
 }
 
+func (o *Output) AsJKS(c *Config) ([]byte, error) {
+
+	var err interface{}
```

Comment:
you dont need to declare err here. You can do it inline as 
var, err := myCode()

Reply:
yes, that declaration is a legancy for the previous changes